From 25ceb980333b5829f21419699378016fc9392e3d Mon Sep 17 00:00:00 2001
From: Angela Cheng <angelaptcheng@gmail.com>
Date: Fri, 21 Aug 2026 02:31:47 +0000
Subject: [PATCH] configuration: schema: add MPQ82D00 PMBus device support

configuration: schema: add MPQ82D00 PMBus device support

Add MPQ82D00 to the legacy and firmware schemas to enable entity-manager
support for the MPQ82D00 PMBus device.

- legacy.json: add "MPQ82D00" to the PMBus device type list
- firmware.json: add "MPQ82D00Firmware" to the firmware type list

The corresponding kernel driver was added in: https://github.com/facebookexternal/openbmc.wiwynn/commit/60337e9b7acd1b126ab9de786a6bfa5a2b263896
---
 schemas/firmware.json | 1 +
 schemas/legacy.json   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/schemas/firmware.json b/schemas/firmware.json
index 39fcb13..44c7c50 100644
--- a/schemas/firmware.json
+++ b/schemas/firmware.json
@@ -54,6 +54,7 @@
                         "MP297XFirmware",
                         "MP5998Firmware",
                         "MP994XFirmware",
+                        "MPQ82D00Firmware",
                         "MPQ87XXFirmware",
                         "RAA22XGen2Firmware",
                         "RAA22XGen3p5Firmware",
diff --git a/schemas/legacy.json b/schemas/legacy.json
index cad0256..40fbbee 100644
--- a/schemas/legacy.json
+++ b/schemas/legacy.json
@@ -1027,6 +1027,7 @@
                         "MP9941",
                         "MP9945",
                         "MP29612",
+                        "MPQ82D00",
                         "MPQ8785",
                         "NCP4200",
                         "PLI1209BC",
--
2.43.0
